runtime/v2/runner/src/main/java/com/walmartlabs/concord/runtime/v2/runner/el/LazyExpressionEvaluator.java

```diff
@@ -162,6 +162,10 @@ public FunctionMapper getFunctionMapper() {
                     .findAny()
                     .map(i -> (RuntimeException)i)
                     .orElse(e);
+        } catch (UserDefinedException e) {
+            throw e;
+        } catch (Exception e) {
+            throw new RuntimeException("Error while evaluating expression '" + expr + "'", e);
```

Comment:
```suggestion
            throw new RuntimeException("while evaluating expression '" + expr + "'", e);
```
just to get rid of repeating "error" in `Error @ ... Error while...`?